components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -41,6 +42,15 @@ enum PremiumStatus {
   ActiveDisconnected,
 };
 
+struct PremiumInfo {
+  // How many credentials in the current batch are remaining
+  uint32 remaining_credential_count;
```

Comment:
Is it technically correct (and understandable) to call it a credential here and not a token?

Reply:
Yes - that's how their referred to elsewhere in the codebase and in the JSON stored in prefs